osf/models/node.py

```diff
@@ -1991,8 +1992,12 @@ def use_as_template(self, auth, changes=None, top_level=True):
             # TODO: explicitly define attributes which may be changed.
         except (AttributeError, KeyError):
             attributes = dict()
-
+        if self.is_deleted:
```

Comment:
That's fine then. `test_cannot_fork_deleted_node` is in `test_node`, can you add a comparable one for nodes/registrations